hail/python/hail/docs/getting_started.rst

```diff
@@ -27,18 +27,20 @@ of the C and C++ standard library:
 For all methods *other than using pip*, you will additionally need
 
 - `Spark 2.4.x <https://www.apache.org/dyn/closer.lua/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz>`_,
-- a C++ compiler, and
-- lz4
+- a C++ compiler,
+- lz4,
+- a compatible BLAS installation such as the Intel MKL or OpenBLAS
```

Comment:
I might say "BLAS and LAPACK" in all of the places you're currently saying "BLAS".